etabotsite/etabotapp/deploy_ui.py

```diff
@@ -0,0 +1,7 @@
+import misc_utils.ng2plus_dist2django as ng2django
+
+ng2django.deploy_ng2plus(
+    '~/projects/etabot/etabot-ui/dist/',
```

Comment:
This is an absolute path which should not appear in production code. Not everybody has `~/project/etabot/etabot-ui/dist` directory on their laptop.